test/simplifyExpression/distributeSearch/distributeSearch.test.js

```diff
@@ -2,7 +2,7 @@ const distributeSearch = require('../../../lib/simplifyExpression/distributeSear
 
 const TestUtil = require('../../TestUtil');
 
-function testDistributeMinus(exprStr, outputStr) {
+function testDistributeResult(exprStr, outputStr) {
```

Comment:
I renamed this function because it seemed more general than just testing the `minus` operation - does that seem okay?